### PR TITLE
Fix Linting: Override with ESLint

### PR DESCRIPTION
**Pull request by subhramit**
I have completed the linting, overriding with ESLint, but this _may_ need a little bit more jugglery with the configuration file (eslint rule disabling) to bypass CI errors. Feel free to co-author commits on this PR as per your requirements.

Refs. https://github.com/LambdaTest/testcafe-browser-provider-lambdatest/pull/62#issuecomment-2396342363

### CODE REVIEW

**Comment by subhramit** [comment]
Okay, CI doesn't seem to be happy, but this time for unrelated reasons!(?)
```
ERROR Unable to open the "lambdatest:Chrome@74.0:Windows 8" browser due to the following error:
Error: Authentication failed. Please assign the correct username and access key to the LT_USERNAME and LT_ACCESS_KEY environment variables.
    at _callee3$ (/home/runner/work/testcafe-browser-provider-lambdatest/testcafe-browser-provider-lambdatest/lib/index.js:109:19)
    at tryCatch (/home/runner/work/testcafe-browser-provider-lambdatest/testcafe-browser-provider-lambdatest/node_modules/@babel/runtime-corejs2/helpers/regeneratorRuntime.js:51:16)
    at Generator.<anonymous> (/home/runner/work/testcafe-browser-provider-lambdatest/testcafe-browser-provider-lambdatest/node_modules/@babel/runtime-corejs2/helpers/regeneratorRuntime.js:139:17)
    at Generator.next (/home/runner/work/testcafe-browser-provider-lambdatest/testcafe-browser-provider-lambdatest/node_modules/@babel/runtime-corejs2/helpers/regeneratorRuntime.js:[8](https://github.com/LambdaTest/testcafe-browser-provider-lambdatest/actions/runs/11237394759/job/31239736809?pr=64#step:8:9)0:21)
    at asyncGeneratorStep (/home/runner/work/testcafe-browser-provider-lambdatest/testcafe-browser-provider-lambdatest/node_modules/@babel/runtime-corejs2/helpers/asyncToGenerator.js:4:17)
    at _next (/home/runner/work/testcafe-browser-provider-lambdatest/testcafe-browser-provider-lambdatest/node_modules/@babel/runtime-corejs2/helpers/asyncToGenerator.js:18:[9](https://github.com/LambdaTest/testcafe-browser-provider-lambdatest/actions/runs/11237394759/job/31239736809?pr=64#step:8:10))
    at /home/runner/work/testcafe-browser-provider-lambdatest/testcafe-browser-provider-lambdatest/node_modules/@babel/runtime-corejs2/helpers/asyncToGenerator.js:23:7
    at new Promise (<anonymous>)
    at new F (/home/runner/work/testcafe-browser-provider-lambdatest/testcafe-browser-provider-lambdatest/node_modules/core-js/library/modules/_export.js:36:28)
    at /home/runner/work/testcafe-browser-provider-lambdatest/testcafe-browser-provider-lambdatest/node_modules/@babel/runtime-corejs2/helpers/asyncToGenerator.js:15:[12](https://github.com/LambdaTest/testcafe-browser-provider-lambdatest/actions/runs/11237394759/job/31239736809?pr=64#step:8:13)
Type "testcafe -h" for help.
```

**Review by harsha509 — COMMENTED** [review]
LGTM!

It would be great if the lint command also gets added in here!
https://github.com/LambdaTest/testcafe-browser-provider-lambdatest/blob/f15fd0ab86e2eb7286593600d3a021c62cb076bd/package.json#L16

**Comment by subhramit** [comment]
> LGTM!
> 
> It would be great if the lint command also gets added in here!
> 
> https://github.com/LambdaTest/testcafe-browser-provider-lambdatest/blob/f15fd0ab86e2eb7286593600d3a021c62cb076bd/package.json#L16

Added.

File: package.json [hunk 1]
```diff
@@ -14,7 +14,8 @@
   ],
   "scripts": {
     "build": "gulp build",
-    "test": "gulp test"
+    "test": "gulp test",
+    "lint": "eslint . --fix"
```
Comment (harsha509) [inline package.json:18]: Great!
But,
running `--fix` will give a different behaviour in CI (which might not be same as in local), we just want to check lint and dev can perform --fix locally!
```suggestion
    "lint": "eslint ."
```

**Review by harsha509 — APPROVED** [review]
Thank you @subhramit !

**Review by Delta456 — APPROVED** [review]

**Pull request merged**